Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
-
+gem 'devise'
+gem 'rexml', '~> 3.2', '>= 3.2.5'
```

Comment:
Did you encounter any problems, so you had to set it that strict?

Reply:
I`ve got an "LoadError - cannot load such file -- rexml/document" and as solution i`ve found that line  above.
I didn`t try using only  "gem 'rexml'".